src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
```

Comment:
The Random instance 'rand' could be a private final field of the ColorSupplier class instead of being created each time the getRandomColor() method is called. This would be more efficient since you would reuse the same Random instance.